config/fcos/v1_6_exp/schema.go

```diff
@@ -33,6 +33,7 @@ type BootDevice struct {
 type BootDeviceLuks struct {
 	Discard   *bool       `yaml:"discard"`
 	Device    *string     `yaml:"device"`
+	Enabled   *bool       `yaml:"enabled"`
```

Comment:
Shouldn't this field be named `cex`? Enabled is ambiguous here.

Reply:
There were few s390x specific parameter  that can be selected under `cex` type. But later those parameter added as default and  hardcoded in the ignition and decided that we can  add in future if there is a requirement eg CIPHER.  That's why the `cex.enabled`. The types is added under v0_6_exp as `cex.enabled`.


```
madhu@Madhus-MacBook-Pro arm64 % ./butane test.bu -o test.ign --pretty
madhu@Madhus-MacBook-Pro arm64 % cat test.bu 
variant: fcos 
version: 1.6.0-experimental
boot_device:
  layout: s390x-zfcp
  luks:
    device: /dev/sda
    enabled: true


madhu@Madhus-MacBook-Pro arm64 % cat test.ign 
{
  "ignition": {
    "version": "3.5.0-experimental"
  },
  "storage": {
    "filesystems": [
      {
        "device": "/dev/mapper/root",
        "format": "xfs",
        "label": "root",
        "wipeFilesystem": true
      }
    ],
    "luks": [
      {
        "cex": {
          "enabled": true
        },
        "device": "/dev/sda4",
        "label": "luks-root",
        "name": "root",
        "wipeVolume": true
      }
    ]
  }
}

```

Reply:
Why is this not cex.enabled like in the 0.6-exp scema?

Reply:
or the luks one

Reply:
Okay, I thought the boot-device sugar we only need `enabled` because it works only with s390x and looks short. Let me try same in fcos too.

Reply:
has this been resolved?